pack/ms.json

```diff
@@ -693,7 +693,7 @@
     "deck_limit": 3,
     "faction_code": "haas-bioroid",
     "faction_cost": 2,
-    "flavor": "End the run. End the run. End the run. End the run.",
+    "flavor": "<strong>End the run.</strong> <strong>End the run.</strong> <strong>End the run.</strong> <strong>End the run.</strong>",
```

Comment:
These are the ones that get smaller and smaller, right? Is `<strong>` appropriate?

Reply:
the dev sheet has these as {b1}End the run.{/b1} {b2}End the run.{/b2} ... {/b4}.

Ignoring them is the other option, but mapping them to a distinct intent is also doable (with corresponding CSS changes).

Again, i dislike the hodgepodge pile of formatting junk.  bleargh.

Reply:
In my opinion, I feel like we shouldn't include them. Software relying on this will produce incorrect styling unless they special case this one, and it feels awkward to add new tags specifically for this card.

Reply:
There is some precedence for resized flavour text:

![NBN: The World is Yours](https://user-images.githubusercontent.com/26557961/180950851-6113ec3b-4bb7-45c6-a5b4-dcf94a86c025.png)

Reply:
we don't do anything different for that flavor text for The World Is Yours in the JSON, though.

Reply:
I think we should either add formatting to replicate the layout of the text on the card (that tools can ignore if its incompatible with them) or not do anything at all. Making the text on Echo bold will just confuse people for the benefit of matching a private document.

Reply:
okee dokee, taking the formatting off of Echo.